README.md

```diff
@@ -1,54 +1,149 @@
+Commodore 64 | ✓ | ? | ?
+Game Boy / GBC | ✓ | ? | ?
+GBA | ✓ | ? | ?
+Genesis | ✓ | ? | ?
+IntelliVision | ✓ | ? | ?
+Master System | ✓ | ? | ?
+N64 | ✓ | ? | ?
+Neo Geo Pocket | ✓ | ? | ?
+NES | ✓ | ? | ?
+PSX | ✓ | ? | ?
+Saturn | ✓ | ? | ?
+SNES | ✓ | ? | ?
+TI-83 | ✓ | ? | ?
+TurboGrafx-16 | ✓ | ? | ?
+Virtual Boy | ✓ | ? | ?
+WonderSwan | ✓ | ? | ?
+ZX Spectrum | ✓ | ? | ?
 
-Release binaries can be found on [on github](http://github.com/TASVideos/BizHawk/releases)
+EmuHawk's frontend also has a comprehensive input mapper, with keyboard and controller support (even simultaneously), hotkeys, and turbo/rapid fire.
 
-[Developer build](https://ci.appveyor.com/project/zeromus/bizhawk-udexo/build/artifacts) of the most recent commit
+In addition to input recording (with TAStudio), you can also record the emulated system's audio and video, either of your own gameplay or of a prerecorded movie (TAS). See the *Usage (advanced)* section below for more info about the many debugging and recording features.
 
-Never mix different versions of BizHawk. Keep each version in its own folder.
+Most of the frontend can be controlled with Lua scripts (only on Windows for now). These scripts can draw overlays for casual play or for RTA speedruns, change the running game to add complex cheats, assist TASers and romhackers in debugging a game, and more.
 
-## Supported Systems
+## Installing — Windows 7/8.1/10
 
- * Nintendo Entertainment System / Famicom / Famicom Disk System (NES/FDS)
- * Super Nintendo (SNES)
- * Nintendo 64
- * Game Boy, Game Boy Color, and Super Game Boy
- * Game Boy Advance
- * Sony PlayStation
- * Sega Master System, Game Gear, and SG-1000
- * Sega Genesis / Sega-CD
- * Sega Saturn
- * PC-Engine (TurboGrafx-16) / CD-ROM & SuperGrafx 
- * Atari 2600
- * Atari 7800
- * Atari Lynx
- * ColecoVision
- * TI-83 Calculator
- * Wonderswan and Wonderswan Color 
- * Apple II
- * Commodore 64
- * Sinclair ZX Spectrum
+Released binaries can be found right here on GitHub:
 
-## Resources
+[![Windows | binaries](https://img.shields.io/badge/Windows-binaries-%230078D6.svg?logo=windows&logoColor=0078D6&style=popout)](https://github.com/TASVideos/BizHawk/releases/latest)
 
-[BizHawk homepage](http://tasvideos.org/Bizhawk.html) 
+Click `BizHawk-<version>.zip` to download it. Also note the changelog, the full version of which is [on tasvideos.org](http://tasvideos.org/Bizhawk/ReleaseHistory.html). **Don't mix different versions** of BizHawk, keep each version in its own folder.
 
-[FAQ](http://tasvideos.org/Bizhawk/FAQ.html) - Frequently Asked Questions / Troubleshooting
+Before you start (by running `EmuHawk.exe`), make sure you have the Windows-only prerequisites installed: .NET Framework 4.6.1; Visual C++ 2010 SP1, 2012, and 2015; and Direct3D 9. If you have a few Steam games, chances are these are already installed, otherwise you can get them all at once with [this program](https://github.com/TASVideos/BizHawk-Prereqs/releases/latest).
```

Comment:
I see you decided to provide textual explanations. It's fine, but all the critical links and notes should be listed together, and then you can put all the details as text.

Reply:
The reason the prereq installer was buried in prose was that I wanted the "if you use Steam" thing to come first. I've removed that and rewritten the paragraph.